src/connector/nvme/nvme_helper.go

```diff
@@ -0,0 +1,31 @@
+package nvme
+
+import (
+	"connector"
+	"time"
+	"utils/log"
+)
+
+func tryDisConnectVolume(tgtLunWWN string, checkDeviceAvailable bool) error {
+	device, err := connector.GetDevice(nil, tgtLunWWN, checkDeviceAvailable)
+	if err != nil {
+		log.Warningf("Get device of WWN %s error: %v", tgtLunWWN, err)
+		return err
+	}
+
+	multiPathName, err := connector.RemoveNvmeFcDevice(device)
+	if err != nil {
+		log.Errorf("Remove device %s error: %v", device, err)
+		return err
+	}
+
+	if multiPathName != "" {
```

Comment:
If this is multipath device, then the device flush attempt has already been done